src/d3a/d3a_core/sim_results/endpoint_buffer.py

```diff
@@ -99,7 +103,8 @@ def generate_json_report(self):
             "progress_info": self.simulation_progress,
             "device_statistics": self.device_statistics.device_stats_time_str,
             "energy_trade_profile": self.file_export_endpoints.traded_energy_profile,
-            "kpi": self.kpi.performance_indices
+            "kpi": self.kpi.performance_indices,
+            "baseline_peak_energy": self.area_throughput_stats.results
```

Comment:
Please rename the same here too. Thanks
```suggestion
            "area_throughput_stats": self.area_throughput_stats.results
```